Corona-Warn-App/src/main/res/values/srs_submission_strings.xml

```diff
@@ -11,4 +11,49 @@
+    <!-- XTXT: Submission consent help others warn other users -->
+    <string name="srs_help_others_warn_other_users_text">Es werden auch Nutzer gewarnt, die zeitgleich mit Ihnen an denselben Events oder Orten eingecheckt waren.</string>
+    <!-- XHED: Submission consent privacy card title -->
+    <string name="srs_privacy_card_title_text">Ihr Einverständnis</string>
+    <!-- XTXT: Submission consent privacy card subtitle -->
+    <string name="srs_privacy_card_subtitle_text">Durch Antippen von „Einverstanden“ willigen Sie in folgende Schritte ein:</string>
+    <!-- XTXT: Submission consent privacy card first bulletpoint -->
+    <string name="srs_privacy_card_first_bulletpoint_text">Die App teilt Ihr positives Testergebnis, um Nutzer, denen Sie begegnet sind, zu warnen. Dies betrifft auch Nutzer der Corona-Warn-App, die zeitgleich am selben Event oder Ort wie Sie eingecheckt waren.</string>
+    <!-- XTXT: Submission consent privacy card second bulletpoint -->
+    <string name="srs_privacy_card_second_bulletpoint_text">Ihre Angaben zur Art des Tests werden anonym statistisch ausgewertet.</string>
+    <!-- XTXT: Submission consent privacy card third bulletpoint -->
+    <string name="srs_privacy_card_third_bulletpoint_text">Wenn Sie zusätzlich Angaben zum Beginn Ihrer Symptome machen, werden auch diese geteilt.</string>
+    <!-- XTXT: Submission consent first bulletpoint -->
+    <string name="srs_submission_consent_first_bulletpoint_text">Ihre Identität bleibt geheim. Andere Nutzer erfahren nicht, wer sein Testergebnis geteilt hat.</string>
+    <!-- XTXT: Submission consent second bulletpoint -->
+    <string name="srs_submission_consent_second_bulletpoint_text">Unter „Meine Check-ins" können Sie Ihre Events und Orte einsehen, deren eingecheckte Gäste gewarnt werden. Sie können einzelne Check-ins auch löschen und so von der Warnung ausschließen.</string>
+    <!-- XTXT: Submission consent third bulletpoint -->
+    <string name="srs_submission_consent_third_bulletpoint_text">Sie können Ihr Einverständnis abgeben, wenn Sie mindestens 16 Jahre alt sind.</string>
+    <!-- XTXT: Submission consent next step submission -->
+    <string name="srs_next_step_submission_text">Erteilen Sie im nächsten Schritt Ihre Erlaubnis zum Zugriff auf die Zufalls-IDs.</string>
+    <!-- XHED: Submission consent request dialog title -->
+    <string name="srs_submission_consent_request_dialog_title">Zufalls-IDs über Corona-Warn-App teilen?</string>
+    <!-- XTXT: Submission consent request dialog body -->
+    <string name="srs_submission_consent_request_dialog_body">Deine Zufallskennungen der letzten 14 Tage werden dazu genutzt, andere darüber zu benachrichtigen, dass du in ihrer Nähe gewesen bist und sie daher möglicherweise in Kontakt mit einer Person gekommen sind, die positiv auf COVID-19 getestet worden ist.\n\nDeine Identität wird niemals mit anderen Personen geteilt.</string>
```

Comment:
Are we changing from "Sie" to "Du" here because the other content on the dialog comes from Google & also uses "Du"?

Reply:
@Ein-Tim  Text is not final yet, most probably it will change in the next days